pkg/qos/REAMD.md

```diff
@@ -0,0 +1,24 @@
+# Overview
+
+This folder holds some sample plugin implementations based on [QoS
+(Quality of Service) class](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/)
+of Pods.
+
+## Maturity Level
+
+<!-- Check one of the values: Sample, Alpha, Beta, GA -->
+
+- [x] 💡 Sample (for demonstrating and inspiring purpose)
+- [ ] 👶 Alpha (used in companies for pilot projects)
+- [ ] 👦 Beta (used in companies, developed actively and ready for production workloads)
```

Comment:
I would drop the "ready for production workloads" here. Or is this similar to k/k?

Reply:
I'd say this doesn't have the exact and strict definition of k/k. We can refine the wordings later. What I want to highlight is the plugins should come from **real requirements** of companies, and the maturity level is in accordance of their usage. But I cannot give a detailed checklist for one to promote to another at this moment.